Userland/Libraries/LibWeb/Layout/GridFormattingContext.cpp

```diff
@@ -1347,6 +1347,24 @@ void GridFormattingContext::determine_grid_container_height()
     m_automatic_content_height = total_y;
 }
 
+void GridFormattingContext::resolve_grid_item_widths()
+{
+    for (auto& item : m_grid_items) {
+        CSSPixels containing_block_width = containing_block_size_for_item(item, GridDimension::Column);
+
+        auto border_left = item.box().computed_values().border_left().width;
+        auto border_right = item.box().computed_values().border_right().width;
+
+        auto& box_state = m_state.get_mutable(item.box());
+        box_state.border_left = border_left;
+        box_state.border_right = border_right;
+
+        auto computed_width = item.box().computed_values().width();
```

Comment:
I would recommend always doing something like this:
```cpp
auto const& computed_width = ...
```
That way, we don't make a temporary copy of the `CSS::Size` which may incur the cost of ref-count manipulation, etc.